cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -160,21 +163,24 @@ private void setDefaultRegulatingControl(CgmesRegulatingControlForStaticVarCompe
         // We try to keep data from original regulating control if available.
         // Even if the regulating control was disabled it may have defined a valid target value
         if (RegulatingControlMapping.isControlModeVoltage(rc.defaultRegulationMode.toLowerCase())) {
-            regulationMode = onlyReactivePowerReg ? StaticVarCompensator.RegulationMode.OFF : StaticVarCompensator.RegulationMode.VOLTAGE;
+            regulationMode = onlyReactivePowerReg ? null : StaticVarCompensator.RegulationMode.VOLTAGE;
             targetVoltage = isValidVoltageFromRegulatingControl(control) ? control.targetValue : rc.defaultTargetVoltage;
         } else if (isControlModeReactivePower(rc.defaultRegulationMode.toLowerCase())) {
             regulationMode = StaticVarCompensator.RegulationMode.REACTIVE_POWER;
             targetReactivePower = isValidReactivePowerFromRegulatingControl(control) ? control.targetValue : rc.defaultTargetReactivePower;
         } else {
             context.fixed("SVCControlMode", () -> String.format("Invalid control mode for static var compensator %s. Regulating control is disabled", svc.getId()));
-            regulationMode = StaticVarCompensator.RegulationMode.OFF;
+            regulationMode = null;
         }
 
         svc.setVoltageSetpoint(targetVoltage);
         svc.setReactivePowerSetpoint(targetReactivePower);
         if (rc.controlEnabled) {
             svc.setRegulationMode(regulationMode);
         }
+        if (regulationMode == null) {
+            svc.setRegulating(false);
+        }
     }
 
     // SVCControlMode and voltageSetPoint are optional in Cgmes 3.0
```

Comment:
Does this comment imply that they are mandatory in CGMES 2.4.15? I will check out of curiosity.